docs/source/reference/training-guide.rst

```diff
@@ -37,6 +37,27 @@ Example configuration:
       A100-80GB:1  
       H100:1  
 
+Use EFA
+~~~~~~~
+Elastic Fabric Adapter (EFA) is a network interface for Amazon EC2 instances that enables users to run applications requiring high levels of inter-node communications at scale on AWS. You can enable EFA on AWS HyperPod/EKS clusters with a simple additional setting in your SkyPilot YAML.
+
+Example configuration:
+
+.. code-block:: yaml
+
+  config:
+    kubernetes:
+      pod_config:
+        spec:
+          containers:
+          - resources:
+              limits:
+                vpc.amazonaws.com/efa: 4
+              requests:
+                vpc.amazonaws.com/efa: 4
+
+See `EFA example <https://docs.skypilot.co/en/latest/examples/training/aws_efa.html>`_ for more details.
```

Comment:
minor: Use `:ref:` isntead of link

Reply:
Updated

Reply:
The generated md files do not have label to refer to, and I updated the link to the GitHub directory. Should we update the `generate_examples.py` to generate reference label for each md file?